src/modules/ChannelSettings/context/ChannelSettingsProvider.tsx

```diff
@@ -139,7 +137,5 @@ const ChannelSettingsProvider: React.FC<ChannelSettingsContextProps> = (props: C
   );
 };
 
-export type UseChannelSettingsType = () => ChannelSettingsProviderInterface;
-const useChannelSettingsContext: UseChannelSettingsType = () => React.useContext(ChannelSettingsContext);
-
+const useChannelSettingsContext = () => React.useContext(ChannelSettingsContext);
```

Comment:
Doesn't need to define the types, because it sets automatically?

Reply:
Yes